table.csv

```diff
@@ -35,6 +45,7 @@ multiaddr,          ,                         0x32
 multibase,          ,                         0x33
 
 multihashes,,
+identity,           raw binary,               0x0
```

Comment:
Didn't we have a code for this one already?

Reply:
We defined it [here](https://github.com/multiformats/multihash/blob/master/hashtable.csv) but never added that definition here.